CFC_DataCollector/trip_generator/input.json

```diff
@@ -1,15 +0,0 @@
-{
-	"radius" : "1",
```

Comment:
We weren't supposed to upload json files which included secret keys :) It is fine to upload samples of json files to indicate the format, and in fact, I have several existing one (keys.json.sample, or config.json.sample).